### PR TITLE
Feature/mahendra

### PR DESCRIPTION
**Pull request by Mahendrarrao**
Changes:

- Added BLOCKLY library
- Created student1.html
- Modified student1.html
- Created student2.html

Current status:
Student1.html - ready for test
student2.html - blank

### CODE REVIEW

File: SER515Project_Team3/WebContent/student1.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Student | Grade 1</title>
+  <script src="./lib/blockly-master/blockly_compressed.js"></script>
+  <script src="./lib/blockly-master/blocks_compressed.js"></script>
+  <script src="./lib/blockly-master/msg/js/en.js"></script>
+  <style>
```
Comment (asing225) [inline SER515Project_Team3/WebContent/student1.html]: Can we add this in a .css file under css directory?

File: SER515Project_Team3/WebContent/student1.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <block type="math_arithmetic"></block>
+    </category>
+    <sep></sep>
+  </xml>
+
+  <script>
+    var demoWorkspace = Blockly.inject('blocklyDiv',
+        {media: './lib/blockly-master/media/',
+         toolbox: document.getElementById('toolbox')});
+  </script>
```
Comment (asing225) [inline SER515Project_Team3/WebContent/student1.html]: Similarly this under js directory?

File: SER515Project_Team3/WebContent/student1.html (hunk 1 above)
Comment (Mahendrarrao) [reply SER515Project_Team3/WebContent/student1.html]: This is possible.

File: SER515Project_Team3/WebContent/student1.html (hunk 2 above)
Comment (Mahendrarrao) [reply SER515Project_Team3/WebContent/student1.html]: Let me check on this.

**Pull request merged**